plugin/tagbar-markdown.vim

```diff
@@ -0,0 +1,57 @@
+
+if exists("tagbar_markdown")
+    finish
+endif
+let tagbar_markdown = 1
+
+
+if (!exists('g:markdown2ctags'))
+    let g:markdown2ctags = '~/.vim/bundle/markdown2ctags/markdown2ctags.py'
```

Comment:
Oh, and while it's longer, I prefer to prefix my global variables more to help avoid collisions... so `g:tagbar_markdown_markdown2ctags` would be more appropriate.